src/content/en/2021/css.md

```diff
@@ -5,17 +5,18 @@ description: CSS chapter of the 2021 Web Almanac covering trends, changes, and p
 authors: [meyerweb, GeekBoySupreme]
 reviewers: [svgeesus, j9t, estelle, bkardell, argyleink, LeaVerou]
 analysts: [rviscomi]
-editors: []
+editors: [shantsis]
 translators: []
+meyerweb_bio:
+GeekBoySupreme_bio:
```

Comment:
@meyerweb / @GeekBoySupreme can we have a short bio for yourselves?

They are shown at the bottom of the chapter. See here for last year's examples: https://almanac.httparchive.org/en/2020/css#authors

Reply:
Ping @meyerweb @GeekBoySupreme

Reply:
I've added a bio for @GeekBoySupreme based on his website to allow this to be merged:

```
Shuvam is a designer, <a hreflang="en" href="https://www.behance.net/shuvammanna">doodler</a>, <a hreflang="en" href="https://distortedaura.wordpress.com/">writer</a>, <a hreflang="en" href="https://www.instagram.com/the_distorted_aura/">shutterbug</a> and a <a hreflang="en" href="https://github.com/GeekBoySupreme">software tinkerer</a>. He's currently designing at <a hreflang="en" href="https://winuall.com">Winuall</a> and Indie-Hacking, working on Projects such as <a hreflang="en" href="https://tabtrum.live">TabTrum</a>.
results: https://docs.google.com/spreadsheets/d/12vQIA0xsC5Jr3J9Sh03AcAvgFjMAmP1xSS6Tjai9LF0/
```

This can be changed later if you want something different.